setup.py

```diff
@@ -13,24 +13,51 @@
 with open(os.path.join(BASE_DIR, "deflate.c"), "r") as src:
     version = re.match(r'.*#define MODULE_VERSION "(.*?)"', src.read(), re.S).group(1)
 
-cpu = "arm" if platform.machine().startswith("arm") else "x86"
+prefer_system_libdeflate = not bool(os.environ.get("USE_BUNDLED_DEFLATE"))
```

Comment:
`USE_BUNDLED_DEFLATE=0` will use the bundled libdeflate, which seems counterintuitive, especially if something is trying to set that env to a true or false value based on a condition (rather than setting/not setting). I think having a list of true or false values and some documentation will take care of this.

Reply:
that's basically my quick and dirty code @bket took from borgbackup.

we only support not setting it or setting it to =yes (or in fact =anything, which all evals to true).

i agree that true / false / True / False / yes / no / 0 / 1 (anything else?) would be cleaner.

Reply:
made it more clean/flexible now.